mainlyetcetera/travel_tracker
# Feature/setup data for user dashboard

**What does this PR do?**
- adds a complete test suite for the `Traveler` class, setting up for the ashboard setup of Iteration 1
**Is this a feature, a fix, or a refactor?**
- feature
**Where should the reviewer start?**
`test/Traveler-test.js`
**How should this be tested?**
- pull down branch (not to main)
- run `npm run test`
- observe that `.beAssigned()`, a `Trip` method, not yet being a function
**This pr is related to which issues?**
#20 #3
